apps/fipsinstall.c

```diff
@@ -323,8 +329,10 @@ static int write_config_fips_section(BIO *out, const char *section,
                       opts->tls_prf_ems_check ? "1" : "0") <= 0
         || BIO_printf(out, "%s = %s\n", OSSL_PROV_PARAM_NO_SHORT_MAC,
                       opts->no_short_mac ? "1" : "0") <= 0
-        || BIO_printf(out, "%s = %s\n", OSSL_PROV_PARAM_DRBG_TRUNC_DIGEST,
+        || BIO_printf(out, "%s = %s\n", OSSL_PROV_FIPS_PARAM_DRBG_TRUNC_DIGEST,
```

Comment:
Was this simply incorrect originally?

Reply:
Yes, although both just happen to be defined to the same string.